### ISSUE TITLE
yarn outdated 

### ISSUE THREAD

**Issue by DennisBecker** [yarnpkg/yarn#1768]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
 yarn outdated                                                                                                                                                           
yarn outdated v0.16.1
error An unexpected error occured, please open a bug report with the information provided in "/Users/becker/git/app-web/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/outdated for documentation about this command.

[yarn-error.log.txt](https://github.com/yarnpkg/yarn/files/583015/yarn-error.log.txt)

**If the current behavior is a bug, please provide the steps to reproduce.**
run ```yarn outdated```

**What is the expected behavior?**
yarn outdated shows outdated packages if any is outdated

**Please mention your node.js, yarn and operating system version.**
node.js 6.7
yarn 0.16.1
MacOS Sierra

**Comment by ldabiralai** [comment]
This is fixed as of #1276, new version needs to be released. In the meantime I believe you can install the nightly build of yarn using:

```
wget https://yarnpkg.com/install.sh
chmod +x install.sh
install.sh --nightly
```

**Comment by DennisBecker** [comment]
Okay, then I have to wait until it is published in npm. Thank you!
